cmd/analyze/main.go

```diff
@@ -33,6 +34,7 @@ var (
 	customSandbox       = flag.String("sandbox-image", "", "override default dynamic analysis sandbox with custom image")
 	customAnalysisCmd   = flag.String("analysis-command", "", "override default dynamic analysis script path (use with custom sandbox image)")
 	listModes           = flag.Bool("list-modes", false, "prints out a list of available analysis modes")
+	features            = flag.String("feature-flags", "", "override default feature flag settings")
```

Comment:
Can we have a way to print out a help list of settable flags?

Reply:
Yes, I will add a follow up PR.